packages/ui-components/src/components/SearchInput/SearchInput.component.tsx

```diff
@@ -207,7 +207,7 @@ export const SearchInput: React.FC<SearchInputProps> = ({
           {...props}
         />
         <div className={getIconWrapperStyles(variant)}>
-          {clear && val?.length && (
+          {clear && val?.length ? (
```

Comment:
normally it is better to be more explicit when checking for length for example if you do it like `val?.length > 0` then you can keep your refactoring rather than converting it back to conditional operator.

```clear && val?.length > 0 && (```

Reply:
or you can also cast it to boolean like `!!val?.length`

Reply:
Sure, that works too